src/main/java/org/terasology/hunger/HungerAuthoritySystem.java

```diff
@@ -82,26 +84,46 @@
     @In
     private Time time;
 
+    /**
+     * Reference to DelayManager, used for addPeriodicAction.
+     */
+    @In
+    private DelayManager delayManager;
+    /**
+     * The interval (in milliseconds) at which healthDecreaseAmount (above) is applied to the component.
+     */
+    public int healthDecreaseInterval = 30000;
+
+    public static final String HUNGER_DAMAGE_ACTION_ID = "Hunger Damage";
     private boolean destroyDrink = false;
 
+    public void postBegin(){
+        boolean processedOnce = false;
+        for(EntityRef entity : entityManager.getEntitiesWith(WorldComponent.class)) {
+            if (!processedOnce) {
```

Comment:
I have used a very basic way of doing what Cervator suggested. 
Any better/efficient way of doing this is appreciated.

Reply:
This works fine :-)

Only obvious enhancement I can think of is adding more log detail, but that's somewhat of a waste since we don't expect it'll ever happen.

We have a telemetry system hooked up, in the ideal work it would be enabled by default and we would configure alerts that would trigger if this log entry ever occurred anywhere. Then at *that* point we could expand on the logging to troubleshoot further.